src/plugins/guile/weechat-guile-api.c

```diff
@@ -24,6 +24,8 @@
 
 #include <libguile.h>
 #include <time.h>
+#include <stdio.h>
+#include <string.h>
```

Comment:
Why are these includes now required?

Reply:
@flashcode These files already needed these includes, I suspect they were included somewhere in guile before and now are not.
```
[ 70%] Building C object src/plugins/guile/CMakeFiles/guile.dir/weechat-guile.c.o
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:130:19: error: implicitly declaring library function 'strdup' with type 'char *(const char *)' [-Werror,-Wimplicit-function-declaration]
    temp_buffer = strdup (*guile_buffer_output);
                  ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:130:19: note: include the header <string.h> or explicitly provide a declaration for 'strdup'
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:150:30: error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
                length = 1 + strlen (temp_buffer) + 1;
                             ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:150:30: note: include the header <string.h> or explicitly provide a declaration for 'strlen'
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:154:21: error: implicitly declaring library function 'snprintf' with type 'int (char *, unsigned long, const char *, ...)' [-Werror,-Wimplicit-function-declaration]
                    snprintf (command, length, "%c%s",
                    ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:154:21: note: include the header <stdio.h> or explicitly provide a declaration for 'snprintf'
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:307:13: error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
        if (strcmp (type_values, WEECHAT_HASHTABLE_STRING) == 0)
            ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:307:13: note: include the header <string.h> or explicitly provide a declaration for 'strcmp'
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:857:17: error: implicitly declaring library function 'strncmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
            if (strncmp (ptr_name, "-q ", 3) == 0)
                ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:857:17: note: include the header <string.h> or explicitly provide a declaration for 'strncmp'
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:1191:5: error: implicitly declaring library function 'memcpy' with type 'void *(void *, const void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
    memcpy (data2, data + start, count);
    ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:1191:5: note: include the header <string.h> or explicitly provide a declaration for 'memcpy'
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:1195:27: error: implicitly declaring library function 'strchr' with type 'char *(const char *, int)' [-Werror,-Wimplicit-function-declaration]
    while ((ptr_newline = strchr (ptr_data, '\n')) != NULL)
                          ^
/tmp/SBo/weechat-2.7/src/plugins/guile/weechat-guile.c:1195:27: note: include the header <string.h> or explicitly provide a declaration for 'strchr'
7 errors generated.
make[2]: *** [src/plugins/guile/CMakeFiles/guile.dir/build.make:63: src/plugins/guile/CMakeFiles/guile.dir/weechat-guile.c.o] Error 1
make[2]: Leaving directory '/tmp/SBo/weechat-2.7/build'
make[1]: *** [CMakeFiles/Makefile2:1636: src/plugins/guile/CMakeFiles/guile.dir/all] Error 2
make[1]: Leaving directory '/tmp/SBo/weechat-2.7/build'
make: *** [Makefile:152: all] Error 2
```